### ISSUE TITLE
ERR Error in ACL SETUSER modifier

### ISSUE THREAD

**Issue by BurningDog** [oliver006/redis_exporter#851]
**Describe the problem**
When copy/pasting the `ACL SETUSER` command in the README I get this error:

`"ERR Error in ACL SETUSER modifier 'new-password': Syntax error"`

This is because there is a space between the `>` and the password - https://redis.io/commands/acl-setuser/ specifies there should be no space.

The command should read:

```
ACL SETUSER <<<USERNAME>>> +client +ping +info +config|get +cluster|info +slowlog +latency +memory +select +get +scan +xinfo +type +pfcount +strlen +llen +scard +zcard +hlen +xlen +eval allkeys on ><<<PASSWORD>>>
```

**Comment by oliver006** [comment]
Ah, thanks for letting me know. 
Any chance you can open a PR to correct the faulty line in the README? Thanks!

**Comment by oliver006** [comment]
Closed by #852
